kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -146,6 +145,8 @@
       const { updateSection, activeResourcePool, selectAllQuestions } = injectQuizCreation();
       const showConfirmationModal = ref(false);
 
+      const prevRoute = ref({ name: PageNames.EXAM_CREATION_ROOT });
```

Comment:
This can be set to the SectionEditor if that's the way we get here so that we can always go back there whenever we got to the resource selection from there